services/scorecard/scorecard.service.js

```diff
@@ -0,0 +1,41 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  score: Joi.required(),
+}).required()
+
+export default class ScorecardScore extends BaseJsonService {
+  static category = 'analysis'
+
+  static route = { base: 'scorecard', pattern: ':host/:orgName/:repoName' }
+
+  async fetch({ host, orgName, repoName }) {
+    return this._requestJson({
+      schema,
+      url: `https://api.securityscorecards.dev/projects/${host}/${orgName}/${repoName}`,
+    })
+  }
+
+  async handle({ host, orgName, repoName }) {
+    const { score } = await this.fetch({ host, orgName, repoName })
+
+    let clr = ''
+
+    if (score <= 3) {
+      clr = 'red'
+    } else if (score <= 5) {
+      clr = 'orange'
+    } else if (score <= 7) {
+      clr = 'yellow'
+    } else {
+      clr = 'green'
+    }
+
+    return {
+      label: 'scorecard score',
+      message: score,
+      color: clr,
+    }
+  }
```

Comment:
This is all related to rendering concerns and we have a very specific pattern we follow for this.

I'd encourage you to read through our [tutorial](https://github.com/badges/shields/blob/master/doc/TUTORIAL.md) on adding new badges which covers this as well as another piece you are missing which are the example listings that make the badge show up on Shields.io